composer.json

```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": ">=5.0.0"
+        "php": ">=7.0.0"
```

Comment:
dont do this, use ^ or ~  https://github.com/Ocramius/PackageVersions/blob/master/composer.json#L13